examples/tutorials/ordinary_model_example.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>Make this function more universal and put it into batchflow.</p><p> </p><p>And show the right class as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>